index.html

```diff
@@ -12,172 +12,198 @@
       <nav class="container">
         <div class="navbar">
           <div class="my-logo"><a href="#">My Logo</a></div>
+          <ul class="nav-list">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about-myself">About</a></li>
+            <li><a href="#contact">contact</a></li>
+          </ul>
           <div class="toggle-menu">
             <img src="images/humbger.png" alt="toggle menu" />
           </div>
         </div>
       </nav>
     </header>
     <section class="headline container">
-      <div class="heading">
-        <h1>
-          I’m Glory <br />Glad to see<br />
-          you!
-        </h1>
+      <div class="heading heading-one">
+        <h1>I’m Glory <br />Glad to see you!</h1>
       </div>
       <div class="paragraph-one">
-        <p class="paragraph">
+        <p class="head-paragraph paragraph">
           I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
-        <p class="lets-connect"><a href="#">LET’S CONNECT</a></p>
+        <p class="lets-connect header-connect"><a href="#">LET’S CONNECT</a></p>
+      </div>
+      <div class="header-socios">
+        <ul class="social-media-links">
+          <li>
+            <a href="#"><img src="images/twitter.png" alt="twitter" /></a>
+          </li>
```

Comment:
- [ ] You should stick to the design as much as possible (e.g. font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly make sure that the `header` is centred as it is in the Figma design :+1: 

Your Design

![Screenshot from 2022-12-10 16-29-46](https://user-images.githubusercontent.com/80458199/206860503-daaca759-c66f-4290-aafb-7e146b2d6f77.png)


Figma Design
![Screenshot from 2022-12-10 16-33-38](https://user-images.githubusercontent.com/80458199/206860532-74202ff4-b279-4f75-9cb9-635346dda2aa.png)